features/src/main/feature/camel-features.xml

```diff
@@ -919,12 +919,12 @@
         <bundle dependency='true'>mvn:com.google.code.gson/gson/2.10.1</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-gson/${project.version}</bundle>
     </feature>
-<!--    <feature name='camel-guava-eventbus' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${project.version}'>camel-core</feature>-->
-<!--        <bundle dependency='true'>mvn:com.google.guava/guava/${guava-eventbus-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.google.guava/failureaccess/1.0.1</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-guava-eventbus/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-guava-eventbus' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>mvn:com.google.guava/guava/${guava-version}</bundle>
+        <bundle dependency='true'>mvn:com.google.guava/failureaccess/1.0.1</bundle>
```

Comment:
```suggestion
        <bundle dependency='true'>mvn:com.google.guava/failureaccess/1.0.2</bundle>
```
FYI, the version I see from the command `mvn dependency:tree -Dscope=runtime` is more `1.0.2` so let me change it for it